spec/classes/php_spec.rb

```diff
@@ -7,27 +7,50 @@
         facts
       end
 
+      php_cli_package = case facts[:osfamily]
+                        when 'Debian'
+                          case facts[:os]['release']['major']
```

Comment:
can't we use the correct facts here?
`Debian` with version `16.04` is confusing. would prefer a switch between debian and ubuntu here.

Reply:
and thank you for your PR. n1!

Reply:
Then I would suggest splitting these two up completely